src/main/java/org/venice/beachfront/bfapi/geoserver/GeoserverEnvironment.java

```diff
@@ -81,7 +83,7 @@ public void initializeEnvironment() {
 
 		// Check GeoServer Layer
 		{
-			final String layerURL = String.format("%s/rest/layers/%s", getGeoServerBaseUrl(), LAYER_NAME);
+			final String layerURL = String.format("%s/layers/%s.json", getWorkspaceBaseUrl(), LAYER_NAME);
```

Comment:
I don't know how this works.  When working from the workspace, do you have to add the .json?

Reply:
I made a change to set the "Accept:" header instead. 

According to the GeoServer API docs you can indicate your response format with either an "Accept:" header or by appending an extension. (e.g. ".json", ".xml", ".html").
Previously we were doing neither and some responses seemed to be xml while others were json.

Reply:
ah ok, yeah it would be nice to keep it consistent